docs/hello_nextflow/02_hello_world.md

```diff
@@ -599,7 +601,7 @@ Learn how to add in a second process and chain them together.
 
 ---
 
-## 8. Add a second step to the workflow
+## 5. Add a second step to the workflow
```

Comment:
This change is wrong

Reply:
If you accept my recommendation to fold the publishDir bit into the previous output section, I think this becomes "## 7." but obviously we'll have to check once they're all nailed down.